crates/web-sys/src/features/gen_AbortController.rs

```diff
@@ -3,7 +3,11 @@ use super::*;
 use wasm_bindgen::prelude::*;
 #[wasm_bindgen]
 extern "C" {
-    # [wasm_bindgen (extends = :: js_sys :: Object , js_name = AbortController , typescript_type = "AbortController")]
+    #[wasm_bindgen(
+        extends = "::js_sys::Object",
+        js_name = "AbortController",
+        typescript_type = "AbortController"
+    )]
```

Comment:
For anyone wondering how this diff is +55k, here's the answer; `rustfmt` now moves these to new lines.